src/main/java/lotto/util/LottoManager.java

```diff
@@ -0,0 +1,143 @@
+    for (Lotto lotto : lottos) {
+      List<Integer> lottoNumbers = new ArrayList<>(lotto.getLottoNumbers());
+      Collections.sort(lottoNumbers);
+      System.out.println(lottoNumbers);
+    }
+  }
+
+  private UserInputNumbers receiveLottoNumber() {
+    while (true) {
+      System.out.println("\n당첨 번호를 입력해 주세요.");
+      String inputNumbers = Console.readLine();
+      try {
+        String[] numbersArray = inputNumbers.split(",");
+        List<Integer> receivedLottoNumbers = new ArrayList<>();
+        Arrays.stream(numbersArray)
+            .forEach(number -> receivedLottoNumbers.add(Integer.parseInt(number.trim()))); // 공백 제거
+        System.out.println("\n보너스 번호를 입력해 주세요.");
+        int bonusNumber = Integer.parseInt(Console.readLine().trim());
+
+        return new UserInputNumbers(receivedLottoNumbers, List.of(bonusNumber));
+      } catch (IllegalArgumentException e) {
+        System.out.println(e.getMessage());
+      }
+    }
+  }
+
+  private Map<WinningCheck, Integer> calculateWinningResult(List<Lotto> lottos
+      , UserInputNumbers receivedLotto) {
+    Map<WinningCheck, Integer> result = new HashMap<>(); // WinningCheck 객체와 정수를 키와 값으로 Map에 저장
+    for (Lotto lotto : lottos) {
+      int matchCount = lotto.getMatchCount(receivedLotto);
+      boolean bonusMatch = lotto.isBonusMatch(receivedLotto);
+
+      WinningCheck prize = WinningCheck.getPrize(matchCount, bonusMatch);
+      result.put(prize // prize 키와 해당하는 값을 Map에 넣는다. 이미 있다면 덮어쓴다
+          , result.getOrDefault(prize, 0) + 1);
+       /*키(prize)에 해당하는 값을 가져온다. 없다면 default(0)을 반환
+       -> 해당하는 현재의 당첨 수를 가져오고 그 수에 1을 더해서 새로운 당첨 수를 계산함*/
+    }
+    return result;
+  }
+
+  private void printWinningResult(Map<WinningCheck, Integer> result) {
+    System.out.println("\n당첨 통계\n---");
+    for (WinningCheck prize : WinningCheck.values()) {
+      if (prize == WinningCheck.LOSE) {
+        continue;
+      }
+      String bonus = prize.hasBonus() ? ", 보너스 볼 일치" : "";
+      System.out.println(
+          prize.getMatchingCount() + "개 일치" + bonus + " (" + prize.getPrizeAmount() + "원) - "
+              + result.getOrDefault(prize, 0) + "개");
+    }
+  }
+
+  private void printProfitRate(int amount, Map<WinningCheck, Integer> result) {
+    double totalPrize = 0;
+    for (Map.Entry<WinningCheck, Integer> entry : result.entrySet()) {
+      // entry: 맵의 각 항목을 나타내는 객체, .entrySet(): 맵의 모든 항목을 객체의 Set으로 반환
+      WinningCheck prize = entry.getKey(); // enum에 열거된 상금을 가져온다
+      int count = entry.getValue(); // 당첨 횟수를 가져온다.
+      totalPrize += Double.parseDouble(prize.getPrizeAmount()
+          .replace(",", "")) * count; // 문자열을 double로 변환하기 위해 쉼표 제거
+    }
+
+    double profitRate = (totalPrize / amount) * PERCENTAGE_FACTOR;
+    DecimalFormat df = new DecimalFormat("#.##");
+    profitRate = Double.parseDouble(df.format(profitRate)); // 소수 둘째 자리에서 반올림
+
+    System.out.println("총 수익률은 " + profitRate + "%입니다.");
```

Comment:
```suggestion
    System.out.println("총 수익률은 " + String.format("%.1f", profitRate) + "%입니다.");
```
처럼 써줄 수도 있어요! DecimalFormat과 어떤 차이가 있을까요?

Reply:
소수를 반올림하는 방법은 제가 알기로 4가지 정도가 있습니다.
첫 번째는 제가 사용한 DecimalFormat 클래스를 사용한 방법이고(#은 0을 생략합니다) 반환값은 String입니다.
두 번째는 BigDecimal 클래스 setScale() 메서드를 사용합니다. (첫 번째 매개변수는 자릿수, 두 번째는 올림, 내림 설정)
세 번째는 suggest 해주신 String 클래스의 format() 메서드입니다. (%.자리f)
네 번째는 Math 클래스의 round() 메서드

DecimalFormat 클래스와 String.format() 의 차이
DecimalFormat 클래스의 경우 인스턴스 생성이 강제되지만 재사용이 가능합니다.
String.format() 은 정적인 사용이 가능합니다.

DecimalFormat은 더 단순하고 직접적인 포맷팅 방식을 사용해서 더 빠릅니다.